compiler.l

```diff
@@ -548,7 +548,7 @@
 (target js: (set (get global 'require) require))
 (target js: (define run eval))
 
-(target lua: (define load1 load))
+(target lua: (define load1 (or loadstring load)))
```

Comment:
Since Lua 5.1 defines both `load` and `loadstring`, we can't use `(or load loadstring)`. 

Here's what would happen:

```
$ LUMEN_HOST=lua5.1 bin/lumen
> 42
error: bad argument #1 to 'load1' (function expected, got string)
```

http://stackoverflow.com/a/37641309

> Since Lua 5.2, `loadstring` has been replaced by `load`.
> From Lua 5.2 reference manual:
>> Function `loadstring` is deprecated. Use `load` instead; it now accepts string arguments and are exactly equivalent to `loadstring`.